src/test/java/com/selina/lending/LendingServiceApplicationTests.java

```
@@ -17,13 +17,37 @@
 
 package com.selina.lending;
 
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import com.selina.lending.api.controller.CreditCommitmentsController;
+import com.selina.lending.api.controller.DIPController;
+import com.selina.lending.api.controller.ProductController;
+import com.selina.lending.api.controller.QuickQuoteController;
+
 @SpringBootTest
 class LendingServiceApplicationTests {
 
+	@Autowired
```

Small question, why do you do these additional asserts about not being null?

As I see it, in this type of test that runs all the Spring context - we definitely should fail if some bean was injected but was not defined.

---

added NOSONAR instead of assertions